server/views/pages/analytics/protected-characteristics/index.njk

```diff
@@ -13,7 +13,7 @@
 
       {% include "../../../partials/messages.njk" %}
 
-      {% set errorSummary = formsWithErrors["incentive-levels-by-ethnicity"].errorSummary | default(formsWithErrors["incentive-levels-by-age-group"].errorSummary) %}
+      {% set errorSummary = formsWithErrors["incentive-levels-by-ethnicity"].errorSummary | default(formsWithErrors["incentive-levels-by-age-group"].errorSummary) | default(formsWithErrors["incentive-levels-by-religion"].errorSummary) | default(formsWithErrors["incentive-levels-by-disability"].errorSummary) %}
```

Comment:
This is quite a long line - you could split it up by pipe:
```
formsWithErrors["incentive-levels-by-ethnicity"].errorSummary | 
 default(formsWithErrors["incentive-levels-by-age-group"].errorSummary) |
 default(formsWithErrors["incentive-levels-by-religion"].errorSummary)
etc
```

Reply:
yeah the way the form errors are chosen for display is a bit gross. what i plan to do in future (INC-437) is have the router know which graphs (and therefore forms/form IDs) are on the page so it can loop through all forms and show any/all errors more easily.